src/core/config/parent_search.h

```diff
@@ -93,4 +107,29 @@
 #define OPENTHREAD_CONFIG_PARENT_SEARCH_RSS_THRESHOLD -65
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_PARENT_SEARCH_RESELECT_TIMEOUT
+ *
+ * Specifies parent reselect timeout duration in seconds used on FED (FTD child) devices. When an attach attempt to a
+ * neighboring router selected as a potential new parent fails, the same router cannot be selected again until this
+ * timeout expires.
+ *
+ * Applicable only if periodic parent search feature is enabled (see `OPENTHREAD_CONFIG_PARENT_SEARCH_ENABLE`).
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_PARENT_SEARCH_RESELECT_TIMEOUT
+#define OPENTHREAD_CONFIG_PARENT_SEARCH_RESELECT_TIMEOUT (90 * 60)
+#endif
+
+/**
+ * @def OPENTHREAD_CONFIG_PARENT_SEARCH_RSS_MARGIN
+ *
+ * Specifies the RSS margin over the current parent RSS for allowing selection of a neighboring router as a potential
+ * new parent to attach to. Used on FED (FTD child) devices.
```

Comment:
Why not used also on REEDs? (Useful in case a REED isn't going to upgrade to Router any time soon, e.g. in larger networks.)

Reply:
Should be already used by REED as well since I assumed REEDs are also classified as being FTD child (FEDs)?

Maybe I am using the terms differently - do we consider FED and REED as disjoint classes? I assumed REEDs are also FEDs.

Reply:
@abtink  FEDs are FTDs that cannot become Router; and cannot become Leader.  REEDs are FTDs that can upgrade to Router, or become Leader.  Hence my confusion, I think!  So it's better to say "FEDs and REEDs" in this case?

Reply:
Good to know. Thanks. I have been using FED to refer to any "FTD child" (which is then not technically equivalent since it excludes REEDs). I updated the PR to not use FED anymore and use FTD device or FTD child directly.